druid_setup/single/environment/middlemanager.env

```diff
@@ -1,3 +1,5 @@
 druid.indexer.task.baseTaskDir=/data/output/druid/task
```

Comment:
`druid.indexer.task.baseTaskDir` -> `druid_indexer_task_baseTaskDir`

Reply:
I fixed this into master already